model_compression_toolkit/gptq/keras/quantizer/gumbel_rounding/gumbel_softmax.py

```diff
@@ -29,7 +29,8 @@ def sample_gumbel(shape, eps=1e-6) -> tf.Tensor:
     return -tf.math.log(-tf.math.log(u + eps) + eps)
 
 
-def gumbel_softmax(in_pi: tf.Tensor, in_tau: tf.Tensor, in_gumbel: tf.Tensor, eps: float = 1e-6, axis=0) -> tf.Tensor:
+def gumbel_softmax(in_pi: tf.Tensor, in_tau: tf.Tensor, in_gumbel: tf.Tensor, eps: float = 1e-6, axis=0,
+                   gumbel_scale: float = 1.0) -> tf.Tensor:
```

Comment:
Why default values of gumbel_scale are different here and in GumbelConfig init? Can we use the same default value?

Reply:
It is intentional, the default in GumbelConfig is according to the behavior we want a MCT user will get (this is the best value for this parameters according to the tests we ran).
In the function itself I prefer to keep a default neutral value which doesn't affect the computation unless the user passes a specific scaling value.
Hope that makes sense..